backend/tests/local/test-storage-rls.sh

```
@@ -0,0 +1,249 @@
+}
+
+list_count() {
+  local jwt="$1" bucket="$2"
+  curl -sS "$API/storage/buckets/$bucket/objects" -H "Authorization: Bearer $jwt" \
+    | python3 -c 'import sys,json; print(json.load(sys.stdin)["pagination"]["total"])'
+}
+
+upload() {
+  local jwt="$1" bucket="$2" key="$3" content="$4"
+  echo "$content" > /tmp/_rls_$TS.txt
+  local code=$(curl -sS -o /dev/null -w "%{http_code}" -X PUT \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)